### ISSUE TITLE
Demo link dead

### ISSUE THREAD

**Issue by fr0z3nfyr** [cnizzdotcom/cakephp-datatable#38]
Any particular reason you have taken down the [demo link](http://cakephpdatatables.cnizz.com/)?

It is often very useful to just get a very quick look at the examples hosted there. Can you please put up a demo on free github pages if cost of maintenance is an issue.

**Comment by cnizzardini** [comment]
Yeah I moved my cnizz.com domain to a new server and missed this. Thanks for pointing this out.
